### PR TITLE
Add stubs for Flask-JSONRPC

### PR DESCRIPTION
**Pull request by nycholas**

### CODE REVIEW

File: stubs/Flask-JSONRPC/@tests/stubtest_allowlist.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+flask_jsonrpc.JSONRPCView.methods
```
Comment (JelleZijlstra) [inline stubs/Flask-JSONRPC/@tests/stubtest_allowlist.txt:1]: Can you comment on why these allowlists are necessary?

**Review by JelleZijlstra — COMMENTED** [review]
This modules comes with a `py.typed`:

```
In [11]: !ls /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/flask_jsonrpc
__init__.py	app.py		contrib		globals.py	py.typed	types.py	wrappers.py
__pycache__	blueprints.py	exceptions.py	helpers.py	site.py		views.py
```

So I don't think it makes sense to add stubs.

(Adding a few other things I noticed before I found the py.typed.)

File: stubs/Flask-JSONRPC/@tests/stubtest_allowlist.txt (hunk 1 above)
Comment (nycholas) [reply stubs/Flask-JSONRPC/@tests/stubtest_allowlist.txt:1]: It was in the previous release, and yesterday I published a new release.

File: stubs/Flask-JSONRPC/flask_jsonrpc/__init__.pyi [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from .app import JSONRPC as JSONRPC
+from .blueprints import JSONRPCBlueprint as JSONRPCBlueprint
+from .views import JSONRPCView as JSONRPCView
+
+__all__ = ["JSONRPC", "JSONRPCBlueprint", "JSONRPCView"]
```
[the remark this answers is not in the document]
Comment (nycholas) [reply stubs/Flask-JSONRPC/flask_jsonrpc/__init__.pyi:5]: I have wronging tested in the previous release, and not in the actual release, I will correct it.

File: stubs/Flask-JSONRPC/@tests/stubtest_allowlist.txt (hunk 1 above)
Comment (nycholas) [reply stubs/Flask-JSONRPC/@tests/stubtest_allowlist.txt:1]: I see the "Third-party library removal policy" now, I understand.

**Comment by nycholas** [comment]
This PR has been closed because of https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#third-party-library-removal-policy.

**Pull request closed without merging**